src/drt/src/dr/FlexGridGraph_maze.cpp

```diff
@@ -58,7 +58,7 @@ void FlexGridGraph::printExpansion(const FlexWavefrontGrid& currGrid,
       pt,
       currGrid.getCost(),
       currGrid.getPathCost(),
-      currGrid.getLastDir(),
+      int(currGrid.getLastDir()),
```

Comment:
I'm in the process of updating spdlog which should obviate this need.  Are you updating spdlog in this PR?  With the current 1.8.1 this shouldn't be needed.  Likewise elsewhere.

Reply:
Ah, heck, why not. Let me try and get back to you.

Reply:
Oh, I misunderstood the message. Yeah, spdlog is part of NixOS so we're not explicitly controlling its version (It's 1.14.something). I will try a build with 1.8.1. Even then however, I would need a matching `fmt`, so it quickly starts spiraling.